app/views/layouts/_flash.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="grid-row">
   <div class="column-two-thirds">
     <% flash.each do |key, value| %>
-      <div class="govuk-<%= key %>-summary" role="alert" aria-labelledby="<%= key %>-summary-heading" tabindex="-1" data-module="error-summary">
-        <h3 class="govuk-<%= key %>-summary__title" id="<%= key %>-summary-heading"><%= value %></h3>
+      <div class="govuk-<%= key %>-summary" role="alert" aria-labelledby="<%= key %>-summary-heading" tabindex="-1" data-module="error-summary" data-ga-event-form="error">
```

Comment:
Maybe:
`data-ga-event-form="error"
```suggestion
      <div class="govuk-<%= key %>-summary" role="alert" aria-labelledby="<%= key %>-summary-heading" tabindex="-1" data-module="error-summary" data-ga-event-form="<%= key %>">
```

Reply:
This change resolve the above?

Reply:
No, the message will still show in the wrong context.